sdk/core/core-http/package.json

```diff
@@ -27,27 +27,27 @@
     "clientruntime"
   ],
   "main": "./dist/coreHttp.node.js",
-  "module": "./es/lib/coreHttp.js",
-  "types": "./es/lib/coreHttp.d.ts",
+  "module": "./es/src/coreHttp.js",
+  "types": "./es/src/coreHttp.d.ts",
   "files": [
     "dist/**/*.node.js",
     "dist/**/*.node.js.map",
     "dom-shim.d.ts",
-    "es/lib/**/*.js",
-    "es/lib/**/*.js.map",
-    "es/lib/**/*.d.ts",
-    "es/lib/**/*.d.ts.map",
-    "lib/**/*.ts",
+    "es/src/**/*.js",
+    "es/src/**/*.js.map",
+    "es/src/**/*.d.ts",
+    "es/src/**/*.d.ts.map",
+    "src/**/*.ts",
```

Comment:
let's drop this line

Reply:
I removed this line `"src/**/*.ts",`  Is that all?

Reply:
I think we're actually supposed to include the TS files in the package:
https://azure.github.io/azure-sdk/typescript_design.html#ts-config-sourceMap

>  Be sure to include your original TypeScript sources in the package.